### DISCUSSION TITLE
S3 storage problem

### DISCUSSION THREAD

**Discussion by 2beetle** [drakkan/sftpgo#1372] [Q&A]
I used S3 as the storage backend, then added a user, and mapped the S3 folders to the virtual path. When I connected to the server through the sftp protocol, I found that I could list objects but not download them.
![image](https://github.com/drakkan/sftpgo/assets/74488739/45ee8a1b-0746-4698-b488-746a5b221a93)
I never changed the configuration file

**Comment by fntyler** [comment]
Just based on the information that you provided, it is possible that the AWS IAM user that you are using doesn't have sufficient permissions?
